curp/doc/steps.md

```diff
@@ -0,0 +1,121 @@
+The document describes the CURP steps in detail.
+
+Definition:
+1. The request tuple consists of request command and following parameters.
+2. The response tuple consists of return code and following states.
+
+# Normal Operation
+## A. Client Send Propose
+
+1. Client send request <Propose, Cmd, RpcId, TermId, WitnessListVersion> to the master and all witnesses.
```

Comment:
What is the difference between Propose and Cmd?

Reply:
Propose is the RPC type, Cmd is the request itself.